Cargo.toml

```diff
@@ -24,10 +24,13 @@ include = [
 
 [dependencies]
 
-combine = "4.6"
-libc = "0.2"
-time = "0.2"
-byteorder = "1.2"
+byteorder = { version = "1.2", default-features = false }
+log = {version = "0.4.21", default-features = false }
```

Comment:
What does `default-features = false` do, please?

Reply:
This is a standard way of telling the dependencies that we are using them with `no_std`. In most cases, the way those libraries are designed is similar to how I specified in `Cargo.toml`: 
```toml
[features]
default = ["std"]
std = ["dep:time", "dep:libc", "dep:combine"]
```
And so if someone was to include `rbpf` in their project and specified: 
```toml
rbpf = { version = "1.2", default-features = false }
```
Then the `std` feature would be disabled and so the crate behaves in a way that is compatible with `no_std`.
```toml
[dependencies]
riot-wrappers = { version = "0.8", features = [ "set_panic_handler" ] }
rbpf = {path = "../../../../vms/rbpf-for-microcontrollers", default-features = false}

# While currently this example does not use any RIOT modules implemented in
# Rust, that may change; it is best practice for any RIOT application that has
# its own top-level Rust crate to include rust_riotmodules from inside
# RIOTBASE.
rust_riotmodules = { path = "../../sys/rust_riotmodules/" }
```
This is an example of how I included `rbpf` without default features in that example program with RIOT on stm32

Reply:
I can add a comment in the `Cargo.toml` explaining why we don't need the default features